auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -1358,4 +1654,26 @@ public XMLStreamWriter getXmlStreamWriter() {
             return xmlStreamWriter;
         }
     }
+
+    public static class IntegrityResult {
```

Comment:
Javadoc should be added for this class and its methods.

Reply:
Added